### PR TITLE
DAOS-8058 tests: rstrip() output of new_daos cont list-attr

### PR DESCRIPTION
**Pull request by bfaccini**
This is a workaround for already existing workaround for
DAOS-6880/DAOS-7093 (use "daos cont list-attr" command instead
of API/daos_cont_list_attr() plugin in pydaos) !!...
This is required as "new" daos command formatted output adds
trailing spaces (for alignment ??) at the end of attributes names
causing to exceed the number of expected total characters in
class ContainerAttributeTest()::verify_get_attr().
So, as a quick+dirty fix, try to rstrip() each attributes names !

Test-tag: pr, container_attribute
Change-Id: Idc2143c40b2701f0383703e22dbc57e45adbd3f8
Signed-off-by: Bruno Faccini <bruno.faccini@intel.com>

### CODE REVIEW

File: src/tests/ftest/util/daos_utils.py [hunk 1]
```diff
@@ -495,6 +495,14 @@ def container_list_attrs(self, pool, cont, sys_name=None):
         # aa bb
         # attr48
         match = re.findall(r"\n([\S ]+)", self.result.stdout_text)
+        # strip all trailing spaces from new daos cmd output formatting
+        # XXX to be removed when DAOS-7093 will be fixed and API/pydaos
```
[the remark this answers is not in the document]
Comment (shimizukko) [reply src/tests/ftest/util/daos_utils.py:499]: What's XXX?

**Review by shimizukko — APPROVED** [review]

**Review by mjean308 — CHANGES_REQUESTED** [review]
Bruno, it appears that some of  the container attr tests are failing. You may want to verify the change is not causing the failures

**Comment by bfaccini** [comment]
> [Bruno,] it appears that some of the container attr tests are failing. You may want to verify the change is not causing the failures
> 
> [https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-6326/8/tests](url)

@mjean308 Yes this is because I am enforcing all container/attributes tests run with the commit/test tag, and I had realized that and have already push a new commit 16ac480 to address/fix the 5th failure, but the 4 others will need to be addressed by DAOS-8214, as it strongly looks as a control-plane/go issue with new daos command !!...

**Comment by mjmac** [comment]
FYI, I think that #6403 fixes this in a different way by converting the tests to use JSON output from the tool.

**Comment by bfaccini** [comment]
Abandoned in favor of [PR-6403](https://github.com/daos-stack/daos/pull/6403)

**Pull request closed without merging**